src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/RazorMapToDocumentRangeParams.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp
 {
-    internal class RazorMapToDocumentRangeParams
+    internal struct RazorMapToDocumentRangeParams
```

Comment:
Needed to do this for ease of setting up mocks. The alternative is to make this `IEquatable`